addons/website_event_track/models/event_track.py

```diff
@@ -451,12 +455,6 @@ def _read_group_stage_ids(self, stages, domain, order):
         """ Always display all stages """
         return stages.search([], order=order)
 
-    def _synchronize_with_stage(self, stage):
```

Comment:
Then it means we could keep published but unvalidated tracks ? Isn't it leading to tracks being displayed without notice ? As published is used for ACLs it is quite important to keep it updated.

Reply:
The track will be published if the track is linked to a track stage having `is_visible_in_agenda` set to `True`. So it is no longer necessary to update `is_published` when the track stage changes. 

I think that changing the `is_published` automatically can be quiet annoying: Maybe the administrator wants to unpublished an event no matter what the track stage is. If `is_published` is updated when the user moves a track from one stage to another, the user will have to constantly check that the track is configured as they want.

Reply:
Our use case was that we have 200 tracks. Publishing them manually was a PITA that people wanted to remove. That's why we added this synchornize mechanism. I would keep it unless LNA is really sure to remove it. Not blocking currently, but at merge time this should be discussed with LNA when he does its final testing.